<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instrument Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2" colspan="2">
        <widget class="QDockWidget" name="dockWidget_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="features">
          <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
         </property>
         <property name="windowTitle">
          <string>Configuration</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_5">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name Server</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="param_nameserver">
             <property name="text">
              <string>192.168.11.17</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="start_pushButton">
             <property name="text">
              <string>Start Instrument Server</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="filename_pushButton">
             <property name="text">
              <string>Config File</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="param_filename"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDockWidget" name="dockWidget_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="features">
          <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
         </property>
         <property name="windowTitle">
          <string>Instruments</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_6">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
           <item>
            <widget class="QDockWidget" name="clientshell_dockWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="features">
              <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
             </property>
             <property name="windowTitle">
              <string>Instrument Manager Client Console</string>
             </property>
             <widget class="QWidget" name="dockWidgetContents_7"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="messages_dockWidget">
   <property name="windowTitle">
    <string>Server Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="message_box"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="servershell_dockWidget">
   <property name="windowTitle">
    <string>Instrument Server Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionOpen_Instruments_Config">
   <property name="text">
    <string>Open Instruments Config</string>
   </property>
  </action>
  <action name="actionSave_Instruments_Config">
   <property name="text">
    <string>Save Instruments Config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
